### PR TITLE
Slightly better output order in the CLI for login

### PR DESCRIPTION
**Pull request by smarterclayton**
[merge]

### CODE REVIEW

File: pkg/cmd/cli/cmd/loginoptions.go [hunk 1]
```diff
@@ -337,14 +337,13 @@ func (o *LoginOptions) gatherProjectInfo() error {
 			}
 		}
 
-		if current, err := oClient.Projects().Get(namespace); err == nil {
-			o.Project = current.Name
-			fmt.Fprintf(o.Out, "Using project %q.\n", o.Project)
-		} else if !kerrors.IsNotFound(err) && !clientcmd.IsForbidden(err) {
+		current, err := oClient.Projects().Get(namespace)
+		if err != nil {
```
Comment (liggitt) [inline pkg/cmd/cli/cmd/loginoptions.go]: Lost the forbidden/NotFound checks. What's the difference in output in those cases?

Comment (liggitt) [inline pkg/cmd/cli/cmd/loginoptions.go]: Wait, you'll fail to list available projects on login if the new user doesn't have access to the previously current project, right?

Comment (smarterclayton) [inline pkg/cmd/cli/cmd/loginoptions.go]: Hrm, wonder if that's a bad stash between when someone else's changes landed.

Comment (smarterclayton) [inline pkg/cmd/cli/cmd/loginoptions.go]: Fixed

Comment (smarterclayton) [inline pkg/cmd/cli/cmd/loginoptions.go]: This is purely reordering the output to say what you are using last.  Frames the list more cleanly.

Comment (smarterclayton) [inline pkg/cmd/cli/cmd/loginoptions.go]:
```
 oc login -u system:admin
You have access to the following projects and can switch between them with 'oc project <projectname>':

  * default
  * extended-test-cli-start-build-2r0o6
  * foo (current)
  * openshift
  * openshift-infra

Using project "foo".
```

**Pull request merged**